src/main/java/com/varbro/varbro/model/Role.java

```diff
@@ -12,7 +12,8 @@
     private Long id;
     private String role;
 
-    public Role() {}
```

Comment:
to bym zostawił jak było, czyli Role() {} chyba tak się przyjmuje zapis bezparametrowego ?